### DISCUSSION TITLE
surveymarks/09-65-1005-1/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#28] [Announcements]
# surveymarks/09-65-1005-1/

http://localhost:4000/surveymarks/09-65-1005-1/

**Comment by jengalas** [comment]
Posted by jason
March 22, 2018 at 2:14 pm
_Imported from WordPress_

Is there a database for the PDH markers? I find them all the time while out caching but have never found anywhere online where the info can be accessed.

Thanks,
Jason / Team_JLB73

**Reply by jengalas** [reply]
Posted by Zhanna
March 22, 2018 at 2:37 pm
_Imported from WordPress_

I have not found anything specifically for PDH markers. The only site with any PennDOT data that I’ve found is [PennDOT PAMS (Photogrammetry Asset Management System)](https://www.surveycontrol.penndot.pa.gov/home.html) and it is woefully sparse.

If you want to check it out, note that the green markers are PennDOT survey points. If you click “Identify Query” and then click on an area of the map, you’ll see datasheets for the nearby marks appear in a frame at the bottom of the page. Clicking on the datasheet icon will bring up the PDF.

Good luck, and please let me know if you discover anything else!
